src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,122 @@
+            </p>
+            <a
+              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center cursor-pointer ${
+                isMobile ? "text-center" : "text-left"
+              }`}
+              onClick={() => console.log("clicked address")}
+            >
+              260 Daytona Blvd, Daytona <br /> Beach, Florida 32114, United{" "}
+              <br /> States
+            </a>
+            <p
+              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center cursor-pointer ${
+                isMobile ? "text-center" : "text-left"
+              }`}
+              onClick={() => console.log("clicked email")}
+            >
+              Info@experteventsolutions.com
+            </p>
+          </div>
+        </div>
+
+        <div className="flex flex-col gap-[24px]">
+          <p
+            className={`font-text text-paragraph-sm sm:hidden ${
+              isMobile ? "hidden" : "block"
+            }`}
+            onClick={() => console.log("clicked contacts")}
+          >
+            Contacts
+          </p>
+          <p
+            className={`font-text text-paragraph-sm sm:hidden ${
+              isMobile ? "hidden" : "block"
+            }`}
+            onClick={() => console.log("clicked rental conditions")}
+          >
+            Rental Conditions
+          </p>
+
+          <div
+            className={`flex flex-wrap flex-col gap-[10px] sm:content-center ${
+              isMobile ? "content-center" : "content-start"
+            }`}
+          >
+            <p
+              className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center ${
+                isMobile ? "text-center" : "text-left"
+              }`}
+            >
+              Social Media
+            </p>
+            <div className="flex flex-row gap-[10px] ">
+              <img
+                className="cursor-pointer"
+                src={instagram}
+                onClick={() => console.log("clicked instagram")}
+                alt="Instagram"
+              />
+              <img
+                className="cursor-pointer"
+                src={facebook}
+                onClick={() => console.log("clicked facebook")}
+                alt="Facebook"
+              />
+              <img
+                className="cursor-pointer"
+                src={linkedin}
+                onClick={() => console.log("clicked linkedin")}
+                alt="LinkedIn"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <p
+        className={`font-text text-paragraph-sm font-normal leading-[26px] tracking-[-0.02em] text-left sm:text-center sm:text-wrap ${
+          isMobile ? "text-center" : "text-left"
+        }`}
+      >
+        Copyright © 2022. Expert Event Solutions - All rights reserved.
```

Comment:
Neat trick here is to plug in Javascript for the year, ie `(new Date).getFullYear()` so this doesn't need updating every year.